calico-cloud_versioned_docs/version-3.17/release-notes/index.mdx

```diff
@@ -2,13 +2,19 @@
 description: What's new, and why features provide value for upgrading.
 ---
 
-# Release notes
+# Calico Cloud release notes
 
-## Calico Cloud 17.1.0
+## September 11, 2023 (version 17.1.1)
 
-September 5, 2023
+### Bug fixes
+
+* We fixed a problem that caused the user interface to crash when a user attempted to edit a policy.
```

Comment:
Since the release the customer tried out the update and it did not fix the problem. I'm ok with leaving this though.